framework/src/utils/FunctionParserUtils.C

```diff
@@ -0,0 +1,58 @@
+#include "FunctionParserUtils.h"
```

Comment:
You mean the copyright comment?
On Mar 6, 2015 9:20 AM, "Cody Permann" <notifications@github.com> wrote:

> In framework/src/utils/FunctionParserUtils.C
> <https://github.com/idaholab/moose/pull/4777#discussion_r25955750>:
>
> > @@ -0,0 +1,58 @@
> > +#include "FunctionParserUtils.h"
>
> missing MOOSE header here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4777/files#r25955750>.
>